test/libsolidity/syntaxTests/types/mapping/mapping_function_calldata.sol

```diff
@@ -0,0 +1,12 @@
+pragma experimental ABIEncoderV2;
+
+contract test {
+    struct S {
+        mapping (uint => uint) s;
+    }
+    function f(S calldata b) external {
+    }
+}
+// ----
+// TypeError: (121-133): Types containing (nested) mappings can only be parameters or return variables of internal or library functions.
+// TypeError: (121-133): Type struct test.S is only valid in storage because it contains a (nested) mapping.
```

Comment:
Could you please add a calldata testcase containing nested struct like so perhaps?

```
pragma experimental ABIEncoderV2;

contract test {
    struct S {
        T t;
    }
    struct T {
        mapping (uint => uint) k;
    }
    function f(S calldata b) external {
    }
}
```